external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -29,45 +30,36 @@ if(${ENABLE_PCMSolver})
                        -DCMAKE_Fortran_COMPILER=${CMAKE_Fortran_COMPILER}
                        -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_BINDIR=${CMAKE_INSTALL_BINDIR}
+                       -DPYMOD_INSTALL_LIBDIR=${PYMOD_INSTALL_LIBDIR}
                        -DCMAKE_INSTALL_DATADIR=${CMAKE_INSTALL_DATADIR}
                        -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
+                       -DENABLE_psi4=ON
                        -DSTATIC_LIBRARY_ONLY=${_a_only}
+                       -DSHARED_LIBRARY_ONLY=${_so_only}
 #                       -DENABLE_OPENMP=???
                        #-DENABLE_XHOST=???  # option missing
                        #-DENABLE_XHOST=${ENABLE_XHOST}  # option missing
                        #-DBUILD_FPIC=${BUILD_FPIC}  # always fpic'd
                        -DENABLE_GENERIC=${ENABLE_GENERIC}
-
-
-#                  -DENABLE_CXX11_SUPPORT=ON
-#                  -DENABLE_64BIT_INTEGERS=${ENABLE_64BIT_INTEGERS}
+                       -DENABLE_CXX11_SUPPORT=ON
+                       -DENABLE_64BIT_INTEGERS=ON
 #                  -DENABLE_TESTS=OFF
 #                  -DENABLE_LOGGER=OFF
 #                  -DENABLE_TIMER=OFF
 #                  -DBUILD_STANDALONE=OFF
 #                  -DENABLE_FORTRAN_API=OFF
 #                  -DSTATIC_LIBRARY_ONLY=${ENABLE_STATIC}
 #                  -DPYTHON_INTERPRETER=${PYTHON_EXECUTABLE}
```

Comment:
We absolutely can pass through the [python trio](https://github.com/psi4/psi4/blob/master/CMakeLists.txt#L39-L41) (I'm not yet tempted to make a target out of them). Python just fell in the class of haven't-needed-to-deal-with-yet among the other external projects.

By the way, have you considered switching to [FindPythonLibsNew](https://github.com/psi4/psi4/blob/master/cmake/FindPythonLibsNew.cmake) which is reputed to be far more reliable on matching interp/libs versions? It's the python detection used by NumPy. Psi4's is a slightly modified version by pybind11. It's only one find_package() call since FindPythonInterp called internally.

And FYI, Psi4's not using the python library anymore. We only detect the library to get at the python header files that are needed by pybind11.